scripts/Update-PortIndex.ps1

```diff
@@ -80,7 +82,7 @@ Get-Content "$Directory/scripts/ci.baseline.txt" | % {
         return
     }
 
-    if ($_ -match '^([-a-z0-9]+):([-a-z0-9]+)\s*=\s*([a-z]+)\s*$') {
+    if ($_ -match '^([-a-z0-9]+):+([-a-z0-9]+)\s*=\s*([a-z]+)\s*$') {
```

Comment:
why do we want to allow multiple `:`s?

Reply:
There is a line in the current ci.baseline.txt:
```
ryu::arm64-windows
```
(or something like it). Because that is part of a different repo, I am accommodating it here. This doesn't change the results for a "correct" baseline.